clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1436,6 +1436,19 @@ static SmallVector<StringRef, 4> serializeSanitizerKinds(SanitizerSet S) {
   return Values;
 }
 
+static void parseSanitizerWeightedKinds(
+    StringRef FlagName, const std::vector<std::string> &Sanitizers,
+    DiagnosticsEngine &Diags, SanitizerSet &S, SanitizerMaskCutoffs *Cutoffs) {
```

Comment:
We don't need ptrs there `SanitizerMaskCutoffs *Cutoffs` -> `SanitizerMaskCutoffs &Cutoffs`

Reply:
Returning by value and rely on RVO is even better

Reply:
Done